### DISCUSSION TITLE
retouch tool

### DISCUSSION THREAD

**Discussion by blackzerotype** [microsoft/winget-pkgs#131810] [Ideas]
Hi. Can we simplify how the retouch tool work? For example, if we use a brush to remove a blemish can we make it behave just like Lightroom or Photoshop where with just a simple stroke of a brush the blemish will go away? And is it possible to remove the extra step of pressing the CTRL key just to add a brush stroke? I think it would be best for the user to simply use the Healing, Clone tool just as they would work in Lightroom or Photoshop. They just keep on brushing the blemished areas to be removed until their satisfied with the results. I noticed this while I tried using the Retouch tool of Darktable v4.6. Thank you for your immediate response.

**Comment by mdanish-kh** [comment]
This issue should be filed over at https://github.com/darktable-org/darktable/issues to bring it to the publisher's attention. This repository only manages manifest files for packages added to the `winget` tool.
